plumber/asset/utils.py

```diff
@@ -13,6 +13,16 @@ def _hashed(s: str) -> str:
         :_B64_LEN
     ].decode("ascii")
 
+def three(a):
```

Comment:
Looks like the additions in this file are not used anywhere. Can you revert these?

Reply:
sure. can't remember why it was added 😅